generators/newcomponent/index.js

```diff
@@ -1,7 +1,7 @@
-const Generator = require("yeoman-generator"),
-    fileaccess = require("../../helpers/fileaccess");
+import Generator from "yeoman-generator";
+import fileaccess from "../../helpers/fileaccess.js";
```

Comment:
are you sure about the `.js`?

Reply:
I am fine dropping it.

Reply:
Wait, with ES modules I have to provide the file extension: https://nodejs.org/api/esm.html#mandatory-file-extensions